src/main/java/com/greenfox/poker/service/GameService.java

```diff
@@ -1,10 +1,48 @@
 package com.greenfox.poker.service;
 
+import com.greenfox.poker.model.Game;
+import com.greenfox.poker.model.GameState;
+import com.greenfox.poker.repository.GameRepo;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class GameService {
 
+  @Autowired
+  GameRepo gameRepo;
 
+  List<GameState> gameStateList = new ArrayList<>();
 
-}
+  public Game saveGame(Game game){
+    game.setCurrentPlayers(0);
+    gameRepo.save(game);
+    return game;
+  }
+
+  public List<Game> getAllGamesOrderedByBigBlind(){
+    return gameRepo.findAllByOrderByBigBlindDesc();
+  }
+
+  public boolean isGameExist(long id){
+    if (gameRepo.exists(id)) {
+      return true;
+    }
+    return false;
+  }
+
+  public Game getGamebyId(long id){
+    return gameRepo.findOne(id);
+  }
+
+  public GameState getGameState(long id) {
+    for (GameState gameState : gameStateList) {
+      if (id == gameState.getId()) {
+        return gameState;
+      }
+    }
```

Comment:
There should be an easier way to get a Gamestate from a collection other then looping through all the list.
Refactor this one.